packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -230,6 +231,28 @@ describe("Garbage Collection Tests", () => {
 				assert(gc.gcEnabled, "gcEnabled incorrect");
 				assert.equal(gc.latestSummaryGCVersion, 1, "latestSummaryGCVersion incorrect");
 			});
+			it("gcEnforcementMinCreateContainerRuntimeVersion later than persisted value, sweepEnabled false", () => {
+				gc = createGcWithPrivateMembers(
+					{ createContainerRuntimeVersion: "1.2.3", sweepEnabled: true },
+					{ gcEnforcementMinCreateContainerRuntimeVersion: "2.0.0" },
```

Comment:
gcEnforcementMinCreateContainerRuntimeVersion is too wordy IMO :)

Reply:
Open to suggestions.

I like it because it's ugly.  It's undocumented so I went for highest degree of clarity with what it does (without writing a complete sentence)